ocaml/xapi/xapi_globs.ml

```diff
@@ -832,7 +832,7 @@ let xenopsd_queues = ref ([
 
 let default_xenopsd = ref "org.xen.xapi.xenops.xenlight"
 
-let ciphersuites_good_outbound = ref "!EXPORT:RSA+AES128-SHA256"
+let ciphersuites_good_outbound = ref "!EXPORT:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-SHA384:RSA+AES128-SHA256:AES128-SHA:AES256-SHA256"
```

Comment:
Is the order here correct? I think AES256 should be preferred ahead of AES128 everywhere.
The requirements list this order, where AES_256 precedes all of AES_128:
TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384, TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384, TLS_RSA_WITH_AES_256_CBC_SHA256, TLS_RSA_WITH_AES_128_CBC_SHA, TLS_RSA_WITH_AES_128_CBC_SHA256
(although I think preferring SHA256 over SHA is the right thing as done in this commit, so the only thing that might have to be changed here is to move the last AES256-SHA256 earlier.